doc/guides/bluetooth/autopts/autopts-linux.rst

```diff
@@ -344,12 +355,40 @@ Server and client by default will run on localhost address. Run server:
    :width: 700
    :align: center
 
-Run client:
+Testing Zephyr Host Stack on QEMU:
+
+.. code-block::
+
+    # Attach Bluetooth Controller(with H4 is an example)
+    # Note:Baudrate 115200 for bbc microbit boards
+    sudo btattach -B /dev/ttyXXX -S 1000000 -R
+
+    sudo bluez/tools/btproxy -u -z
+
+    python ./autoptsclient-zephyr.py "C:\Users\USER_NAME\Documents\Profile Tuning Suite\PTS_PROJECT\PTS_PROJECT.pqw6" \
+    ~/zephyrproject/build/zephyr/zephyr.elf -i SERVER_IP -l LOCAL_IP
+
+
+Testing Zephyr Host Stack on native posix:
+
+.. code-block::
+
+    # Attach Bluetooth Controller(with H4 is an example)
+    # Note:Baudrate 115200 for bbc microbit boards
+    sudo btattach -B /dev/ttyXXX -S 1000000 -R
+
+    west build -b native_posix zephyr/tests/bluetooth/tester/ -DOVERLAY_CONFIG=overlay-native.conf
+
+    sudo python ./autoptsclient-zephyr.py "C:\Users\USER_NAME\Documents\Profile Tuning Suite\PTS_PROJECT\PTS_PROJECT.pqw6" \
+    ~/zephyrproject/build/zephyr/zephyr.exe -i SERVER_IP -l LOCAL_IP --hci 0
```

Comment:
Please add some level of indentation to show better show that it's part of the same command as well
```suggestion
        ~/zephyrproject/build/zephyr/zephyr.exe -i SERVER_IP -l LOCAL_IP --hci 0
```